update that this is not supported (#8736)
circleci/circleci-docs: jekyll/_cci2/contexts.adoc

[#restrict-a-context-to-a-security-group-or-groups]
=== Restrict a context to a security group or groups

You must be an organization administrator to complete the following task. Note: this is not supported for orgs that integrate with xref:github-apps-integration/[CircleCI's GitHub App].

. Navigate to menu:Organization Settings[Contexts] in the CircleCI web app to see the list of contexts. The default security group is `All members`, and allows all users in the organization to invoke jobs with that context.
. Click the *Create Context* button if you wish to use a new context, or click the name of an existing context (if using an existing context, you will need to remove the `All members` security group before adding a new one).

> . Click the *Add Security Group* (GitHub OAuth App users) or *Add Project Restriction* (GitLab users) button to view the dialog box.

Reviewer: Remove The Gitlab reference here

Author: updated

> . Make your choices in the dialog box and then click the *Add Security Group* or *Add Project Restriction* button to finalize. Contexts will now be restricted to the selections you have made.

Reviewer: Remove the project restrictions as well

[#restrict-a-context-to-a-security-group-or-groups]
=== Restrict a context to a security group or groups

You must be an organization administrator to complete the following task. 

> NOTE: This is not supported for orgs that integrate with xref:github-apps-integration#[CircleCI's GitHub App], xref:gitlab-integration#[GitLab], or Bitbucket.

Reviewer 2: ```suggestion
NOTE: Restricting a context to a security group is supported for orgs integrated through the GitHub OAuth app and **not supported** for orgs that integrate with xref:github-apps-integration#[CircleCI's GitHub App], xref:gitlab-integration#[GitLab], or Bitbucket.
```

. Navigate to menu:Organization Settings[Contexts] in the CircleCI web app to see the list of contexts. The default security group is `All members`, and allows all users in the organization to invoke jobs with that context.
. Click the *Create Context* button if you wish to use a new context, or click the name of an existing context (if using an existing context, you will need to remove the `All members` security group before adding a new one).

> . Click the *Add Security Group* (GitHub OAuth App users) button to view the dialog box.

Reviewer 2: ```suggestion
. Click the *Add Security Group* button to view the dialog box.
```

Reviewer 2: I think we don't need this as it's specified in the banner

[#add-and-remove-users-from-teams-and-groups]
=== Add and remove users from teams and groups

> *GitHub OAuth App users:* CircleCI syncs GitHub team and LDAP groups every few hours. If a user is added or removed from a GitHub team or LDAP group, it will take up to a few hours to update the CircleCI records and remove access to the context.

Reviewer 2: ```suggestion
*GitHub OAuth app users:* CircleCI syncs GitHub team and LDAP groups every few hours. If a user is added or removed from a GitHub team or LDAP group, it will take up to a few hours to update the CircleCI records and remove access to the context.
```

[#add-and-remove-users-from-teams-and-groups]
=== Add and remove users from teams and groups

> *GitHub OAuth app users:* CircleCI syncs GitHub team and LDAP groups every few hours. If a user is added or removed from a GitHub team or LDAP group, it will take up to a few hours to update the CircleCI records and remove access to the context.

Reviewer 2: ```suggestion
*GitHub OAuth app users:* CircleCI syncs GitHub team and LDAP groups every few hours. If a user is added or removed from a GitHub team or LDAP group, it can take up to a few hours to update the CircleCI records and remove access to the context.
```
